Formula/unbound.rb

```diff
@@ -13,11 +13,13 @@ class Unbound < Formula
 
   depends_on "openssl"
   depends_on "libevent"
+  depends_on "expat"
```

Comment:
Ah, I'm almost certain it's because you don't have the Command Line Tools package installed, which is perfectly valid & supported by both us & Apple, but some upstreams struggle with it. Can you remove the `expat` dependency & try this locally:

```ruby
def install
  args = %W[
    --prefix=#{prefix}
    --sysconfdir=#{etc}
    --with-libevent=#{Formula["libevent"].opt_prefix}
    --with-ssl=#{Formula["openssl"].opt_prefix}
  ]
  args << "--with-libexpat=#{MacOS.sdk_path}/usr/include" unless MacOS::CLT.installed?

  system "./configure", *args
  inreplace "doc/example.conf", 'username: "unbound"', 'username: "@@HOMEBREW-UNBOUND-USER@@"'
  system "make", "install"
end
```